docs/README.Windows.md

```diff
@@ -87,12 +87,17 @@ Start the Visual Studio installer and click **Workloads** select
   * `Desktop development with C++`
     * Select the optional component: `Windows 11 SDK (10.0.22621.0)`
   * `WinUI application development` (if compiling for UWP)
-    * Select the optional component: `Universal Windows Platform tools`
-    * Select the optional component: `C++ (v143) Universal Windows Platform tools`
+
+### Visual Studio 2026 install notes
+Start the Visual Studio installer and click **Workloads** select
+* Under **Desktop & Mobile** section select
+  * `Desktop development with C++`
+    * Select the optional component: `Windows 11 SDK (10.0.26100.x)`
+  * `WinUI application development` (if compiling for UWP)
 
 Click in **Individual components** select
 * Under **Compilers, build tools and runtimes** section select
-  * `MSVC v143 - VS 2022 C++ ARM64 build tools (Latest)` (if compiling for ARM64)
+  * `MSVC Build Tools for ARM64/ARM64EC (Latest)` (if compiling for ARM64)
```

Comment:
That option does not exist in the VS 2022 installer, only explicitly spelled out version, like the previous version of the readme had. Should be 1.43 for consistency with official builds/other devs, not 1.41 or 1.42

Is the version-less option new in VS 2026?